dashboard/src/actions/charts.test.tsx

```diff
@@ -89,16 +95,15 @@ const fetchChartsTestCases: IFetchChartsTestCase[] = [
         payload: {
           items: [chartItem],
           page: 1,
-          totalPages: 3,
-        },
+          nextPageToken: "3",
```

Comment:
The pageSize is being set globally. It could be configurable, but right now it is hardcoded to `20`. The current approach just requests chunks of 20 results.
What do you mean by "an item limit" here?

Reply:
With the new API, the request `page_token` and the corresponding `next_page_token` in the response, need to encode the number of items, not number of pages (ie. instead of `nextPageToken: "3"` here, we'll eventually have `nextPageToken: "41"` - so that the next request will start from item 41 (assuming that a similar `pageSize` of 20 is used. The reason we need to do this is so we can calculate the `nextPageToken` for aggregated data where we might only use part of the data from a particular plugin. Anyway, it's just something on my TODO, not relevant for this PR.